src/tests/ftest/util/agent_utils.py

```diff
@@ -26,33 +26,120 @@
 import os
 import time
 import subprocess
-import json
 import getpass
 import socket
 import errno
 import fcntl
 import re
+import yaml
```

Comment:
(pylint-import-error) Unable to import 'yaml'

Reply:
Dependency:
the commit of https://github.com/daos-stack/daos/pull/1415